### PR TITLE
Adds a live preview markdown editor field [7 / 7]

### PR DESCRIPTION
**Pull request by jebw**
### Jira link

HOTT-????

### What?

I have added/removed/altered:

- [x] Added a markdown preview to the govuk markdown field
- [x] Added a webpacker-ised version of the preview javascript

### Why?

I am doing this because:

- The existing UI framework is hard to maintain
- We can get more value for money investing in the new govuk framework
- Its one less tool for the developers to need to understand
- We will be expanding the admin area in the coming months

### Have you? (optional)

- [x] Reviewed view changes with stake holders
- [x] Validated mobile responsive behaviour of view changes

### Deployment risks (optional)

- currently it needs GOVUK_FRONTEND=true so it can be booted in either UI

### Screenshots

Screenshot from 2021-11-15 15-30-19
Screenshot from 2021-11-15 15-24-17
Screenshot from 2021-11-15 15-23-55
Screenshot from 2021-11-15 15-23-21

### CODE REVIEW

File: app/webpacker/javascripts/jquery.autosize.js [hunk 1]
```diff
@@ -0,0 +1,258 @@
+/*!
```
Comment (willfish) [inline app/webpacker/javascripts/jquery.autosize.js:1]: Should we not bundle this as part of our yarn install process?

File: app/webpacker/javascripts/markdown-preview.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import $ from 'jquery';
+import 'jquery.autosize' ;
+
+$(document).ready(function(){
+  var Previewer = {
```
Comment (willfish) [inline app/webpacker/javascripts/markdown-preview.js:5]: I think if this is our code we could do with writing some tests for this.

**Review by willfish — APPROVED** [review]
Comments not blocking. This is awesome

File: app/webpacker/javascripts/jquery.autosize.js (hunk 1 above)
Comment (jebw) [reply app/webpacker/javascripts/jquery.autosize.js:1]: Couldn't figure out upstream - I think really we should be dropping the jQuery code anyway - that can happen later though

File: app/webpacker/javascripts/markdown-preview.js (hunk 2 above)
Comment (jebw) [reply app/webpacker/javascripts/markdown-preview.js:5]: Yeh - I started adding in stimulus and jest but figured that can come later - this is very much the tactical solution

**Pull request merged**